src/patient-registration/patient-registration.component.tsx

```diff
@@ -172,9 +162,10 @@ export const PatientRegistration: React.FC = () => {
       Array.prototype.forEach.call(nameMappings, nameMapping => {
         let name = nameMapping.getAttribute('name');
         let defaultValue = getValueIfItExists(name, 'elementDefaults', templateXmlDoc);
-        initialFormValues[name] = defaultValue ?? '';
+        initialAddressFieldValues[name] = defaultValue ?? '';
       });
 
+      Object.assign(initialFormValues, initialAddressFieldValues);
```

Comment:
Hm can we find a way without mutating this global?

Reply:
This global variable is already been mutated in different places https://github.com/openmrs/openmrs-esm-patient-registration/blob/master/src/patient-registration/patient-registration.component.tsx#L125 and https://github.com/openmrs/openmrs-esm-patient-registration/blob/master/src/patient-registration/patient-registration.component.tsx#L175.

This variable is used to initialise the initial values for formik and I had to mutate it to avoid playing with the `field.value` on the `input` field.

Reply:
Alright, I would prefer not to for many reasons (does not react, mixes up the type, side-effects always lead to potential bugs), but we can leave it like that.